plugins/main/ptp/ptp.go

```diff
@@ -120,26 +127,33 @@ func setupContainerVeth(netns, ifName string, mtu int, pr *types.Result) (string
 	return hostVethName, err
 }
 
-func setupHostVeth(vethName string, ipConf *types.IPConfig) error {
+func setupHostVeth(vethName string, ipamResult *types.Result) error {
 	// hostVeth moved namespaces and may have a new ifindex
 	veth, err := netlink.LinkByName(vethName)
 	if err != nil {
 		return fmt.Errorf("failed to lookup %q: %v", vethName, err)
 	}
 
-	// TODO(eyakubovich): IPv6
+	ipConfig := ipamResult.IP6
+	cidrOnes := 128
+	cidrBits := 128
+	if ipConfig == nil {
```

Comment:
Maybe even a helper function like:

```golang
func getIpConfig(result types.Result) (*types.IPConfig, uint, uint) {
    <stuff>
}
```

which might be a bit clearer in the code.